index.js

```diff
@@ -366,6 +366,7 @@ RedirectableRequest.prototype._processResponse = function (response) {
   // care for methods not known to be safe, […]
   // RFC7231§6.4.2–3: For historical reasons, a user agent MAY change
   // the request method from POST to GET for the subsequent request.
+  var originalRequestMethod = this._options.method;
   if ((statusCode === 301 || statusCode === 302) && this._options.method === "POST" ||
```

Comment:
```suggestion
  var method = this._options.method;
  if ((statusCode === 301 || statusCode === 302) && method === "POST" ||
```
(and throughout)

Reply:
Updated in 9771127024beab790ade8bc39a0d88b7fa61c83e